src/store/modules/layout.ts

```diff
@@ -34,16 +34,15 @@ export enum LayoutMutations {
   setDashboardLayout = 'setDashboardLayout',
   setTradingLayout = 'setTradingLayout',
 }
-
 // Define default layouts
 const DEFAULT_TRADING_LAYOUT: GridItemData[] = [
-  { i: TradeLayout.botControls, x: 0, y: 0, w: 3, h: 3 },
-  { i: TradeLayout.multiPane, x: 0, y: 3, w: 3, h: 7 },
-  { i: TradeLayout.chartView, x: 3, y: 0, w: 9, h: 11 },
-  { i: TradeLayout.tradeDetail, x: 0, y: 11, w: 5, h: 6 },
-  { i: TradeLayout.openTrades, x: 5, y: 11, w: 7, h: 5 },
-  { i: TradeLayout.tradeHistory, x: 5, y: 12, w: 7, h: 6 },
-  { i: TradeLayout.logView, x: 0, y: 16, w: 12, h: 3 },
+  { i: TradeLayout.botControls, x: 0, y: 0, w: 4, h: 3 },
+  { i: TradeLayout.multiPane, x: 0, y: 3, w: 4, h: 28 },
+  { i: TradeLayout.chartView, x: 4, y: 0, w: 8, h: 11 },
```

Comment:
While on the one hand it seems more even now (so in general, i like it), there's a lot fo wasted space in the left hand column, while the chart is now ... kinda smallish (that's why i had the top layout at 3:9 instead of 4:8 in the first place).
Is there any particular reason you think the chart should be smaller than it is? (side note: The remaining fixes to this layout are great!)


The only field in "multi-pane" that benefits from this seems to be "status" - which is probably the one people look at the least - as it only provides some "quick info" data.

Reply:
I generally don't use the chart in frequi that often as it cannot be as realtime as in the exchange itself.
But the reason i have a wider mulitpane is because i seem to have a lot of folating points in my daily stats, which is pretty unreadable while being narrower:

![image](https://user-images.githubusercontent.com/16370656/106249433-a7423700-6212-11eb-8b73-bfbc49f7048d.png)

I'll be looking into something to filter out folating points, and average them up for this reason.

Reply:
hm, there SHOULD be a "formatnumber" (*name might be wrong) that's applied and default-rounds it to 8 decimals.
Search for it in `formatters.ts` ... (i'm sorry, i don't remember the proper name - but it's used in the trades display too).

Reply:
@mads03dk - maybe you can merge the current master branch into this version - i've fixed the "daily" view (in da3460c) now so maybe we can then reduce this to a width of 3 again ... ?

Reply:
one thing i noticed is I don't see closed trades that were closed as loss from stoploss

Reply:
@gacopl Let's not abuse a Pull request for reporting issues please, thanks.

Reply:
@xmatthias yes let's add feature to already bugged part that makes more sense :) wasn't trying to abuse anything just give you or the requester a hint take it or leave it

Reply:
I've been busy lately. I will have a look at [`da3460c`](https://github.com/freqtrade/frequi/commit/da3460ca0f3089ff3a81ff53bf44f1bbd812a858) 

 @gacpol
> one thing i noticed is I don't see closed trades that were closed as loss from stoploss

I have no problem seeing trades closed by stoploss, nor trailing stoploss? Close reason is sent in the same way for all reasons since it's more or less just a DB query
![image](https://user-images.githubusercontent.com/16370656/106901651-52159200-66f8-11eb-945b-18394dae4835.png)
![image](https://user-images.githubusercontent.com/16370656/106901694-5e015400-66f8-11eb-94a7-c972efbd831c.png)

Reply:
ok sorry guys i found in the forrest of trades i could not see it because table sorts by trade id not closed date my stupid ux assumption

anyways thanks for great work both on frequi and this PR will not abuse in future :)